docs-site/docs/user/organization/members/group-and-project-members.md

```diff
@@ -137,10 +139,11 @@ Prerequisites:
 
 To remove a member from a project:
 
-1. From the left sidebar, select **Projects**, and find your project
-2. From the left sidebar, select **Members**
-3. On the right hand side of the row for the member you want to remove, click the `ellipsis` and select **Remove** from the dropdown.
-4. Confirm that you would like to remove the member from the project in the popup by clicking the **OK** button
+1. From the left sidebar, select **Projects**
+2. Select the project
```

Comment:
Below this there is the `Remove a member from a group`, which should also have the wording changed like this.

Reply:
Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d